bot/seasons/easter/traditions.py

```diff
@@ -0,0 +1,33 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'traditions.json'), 'r', encoding="utf8") as f:
+    traditions = json.load(f)
+
+
+class Traditions(commands.Cog):
+    """A cog which allows users to get a random easter tradition or custom from a random country."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(alias='eastercustoms')
```

Comment:
The kwarg here is [`aliases`](https://discordpy.readthedocs.io/en/rewrite/ext/commands/api.html#discord.ext.commands.Command.aliases), and is a list/tuple:

```suggestion
    @commands.command(aliases=('eastercustoms',))
```

Reply:
Committed directly from here.